<launch>
  <include file="$(find turtlebot_bringup)/launch/bringup.launch"/>
  <include file="$(find turtlebot_navigation)/launch/amcl_ldemo.launch"/>
  <include file="$(find turtlebot_rviz_launchers)/launch/view_navigation.launch"/>  
  
  <!-- <node pkg="rip_edu_state" type="run_launch.py" name="run_launch" output="screen"/> -->
  <node pkg="rip_edu_state" type="stop_follower.py" name="stop_follower" output="screen"/>
  
  <node pkg="rip_edu_voice" type="text_to_speech.py" name="text_to_speech" output="screen"/>
  <node pkg="rip_edu_voice" type="speech_to_text.py" name="speech_to_text" output="screen"/>

	<include file="$(find freenect_launch)/launch/freenect.launch"/>
	<node pkg="rip_edu_vision" type="hand_detection.py" name="hand_detection" output="screen"/>
  <!-- <node pkg="rip_edu_vision" type="fake_hand_detection.py" name="hand_detection" output="screen"/> -->

  <node pkg="rip_edu_navigation" type="nav_to_pose.py" name="nav_to_pose" output="screen"/>

  <include file="$(find rip_edu_manipulator)/launch/launch_arm.launch"/>
  
  <node pkg="rip_edu_state" type="run_launch.py" name="run_launch" output="screen"/>

</launch>